mod_ngarn/worker.py

```diff
@@ -76,13 +78,20 @@ class Job:
             next_schedule,
         )
 
+    async def delay(self):
+        delay = math.exp(self.priority)
+        if self.max_delay and delay > self.max_delay:
+            return self.max_delay
+        return delay
```

Comment:
Isn't this also `return max(delay, self.max_delay or delay)`

Reply:
Thank you